.github/workflows/cicd.yml

```
@@ -0,0 +1,165 @@
+
+jobs:
+  run_tests:
+    name: 🧪 테스트 실행 (테스트 제외)
+    if: github.event.action == 'opened' || github.event.action == 'synchronize'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 코드 체크아웃
+        uses: actions/checkout@v4
+
+      - name: JDK 설치
+        uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)